assets/js/src/core/components/element-tree/node/tree-node.tsx

```diff
@@ -26,6 +27,7 @@ export interface TreeNodeProps {
   internalKey: string
   children: TreeNodeProps[]
   level: number
+  permissions: AssetPermissions
```

Comment:
Here we are in the element-tree but only support the `AssetPermissions`- there are some differences between assets and data objects in detail. We need to somehow support all element types.

Reply:
created a `ElementPermission` type to address that issue. This should do the job. The location (inside the `element-api-slice-enhanced.ts` can be changed if its not fitting`)

Reply:
I think this is fine, thanks 👍